emulator/app/src/tests/mctp_util/ctrl_protocol.rs

```diff
@@ -0,0 +1,227 @@
+// Licensed under the Apache-2.0 license
+
+use bitfield::bitfield;
+use zerocopy::{FromBytes, Immutable, IntoBytes, KnownLayout};
+
+pub const MCTP_CTRL_MSG_HDR_SIZE: usize = 2;
+
+#[macro_export]
+macro_rules! set_eid_req {
```

Comment:
These macros feel like they could just be simple functions?

Like `fn set_eid_req(op: u8, eid: u8) -> Vec<u8>`?

Reply:
changed to functions